### ISSUE TITLE
serialization error

### ISSUE THREAD

**Issue by brianfromoregon** [Lycoon/clash-api#21]
When calling:

`War currentWar = clashApi.getCurrentWar(myClan);`

I get
```

kotlinx.serialization.MissingFieldException: Field 'members' is required for type with serial name 'com.lycoon.clashapi.models.war.WarClan', but it was missing at path: $.clan at path: $.clan
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:90)
	at kotlinx.serialization.json.Json.decodeFromString(Json.kt:95)
	at com.lycoon.clashapi.core.ClashAPI.getCurrentWar(ClashAPI.kt:427)
	at com.umpquatechnology.Main.clashCall(Main.java:103)
	at com.umpquatechnology.Main.main(Main.java:27)
Caused by: kotlinx.serialization.MissingFieldException: Field 'members' is required for type with serial name 'com.lycoon.clashapi.models.war.WarClan', but it was missing at path: $.clan
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:90)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeSerializableValue(AbstractDecoder.kt:43)
	at kotlinx.serialization.encoding.AbstractDecoder.decodeNullableSerializableElement(AbstractDecoder.kt:79)
	at com.lycoon.clashapi.models.war.War$$serializer.deserialize(War.kt:5)
	at com.lycoon.clashapi.models.war.War$$serializer.deserialize(War.kt:5)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:70)
	... 4 more
Caused by: kotlinx.serialization.MissingFieldException: Field 'members' is required for type with serial name 'com.lycoon.clashapi.models.war.WarClan', but it was missing
	at kotlinx.serialization.internal.PluginExceptionsKt.throwMissingFieldException(PluginExceptions.kt:20)
	at com.lycoon.clashapi.models.war.WarClan.<init>(WarClan.kt:6)
	at com.lycoon.clashapi.models.war.WarClan$$serializer.deserialize(WarClan.kt:6)
	at com.lycoon.clashapi.models.war.WarClan$$serializer.deserialize(WarClan.kt:6)
	at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:70)
	... 9 more
```

**Comment by brianfromoregon** [comment]
This happens when not in war. Response:

{"state":"notInWar","clan":{"badgeUrls":{"small":"https://api-assets.clashofclans.com/badges/70/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png","large":"https://api-assets.clashofclans.com/badges/512/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png","medium":"https://api-assets.clashofclans.com/badges/200/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png"},"clanLevel":0,"attacks":0,"stars":0,"destructionPercentage":0.0},"opponent":{"badgeUrls":{"small":"https://api-assets.clashofclans.com/badges/70/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png","large":"https://api-assets.clashofclans.com/badges/512/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png","medium":"https://api-assets.clashofclans.com/badges/200/H39b_-WLZGtZVWQ0hqTkE-Tn2AaQnQWy_Iz4yBlvL0M.png"},"clanLevel":0,"attacks":0,"stars":0,"destructionPercentage":0.0}}

**Comment by Lycoon** [comment]
Thanks! Fixed in [3155a13](https://github.com/Lycoon/clash-api/commit/3155a130d7819014827ef6e2dde311fcf67a3881)
